src/Database/Driver/PDODriverTrait.php

```diff
@@ -158,7 +158,7 @@ public function rollbackTransaction()
             return false;
         }
 
-        return $this->_connection->rollback();
+        return $this->_connection->rollBack();
```

Comment:
Existing method casing is already correct.

https://github.com/cakephp/cakephp/blob/62944e3bc04bd62459b3fc43c33e06d245837cb1/src/Database/Connection.php#L544

Reply:
I wonder how that tool thinks otherwise.

Reply:
The _connection seems to be an instance of PDI. So PDO::rollBack() would be correct.